Documents/Documents/Code/Controllers/Filters/ASCFiltersViewController.swift

```diff
@@ -121,10 +141,19 @@ private extension ASCFiltersViewController {
     @objc func cancelBarButtonItemTapped() {
         dismiss(animated: true)
     }
+
+    @objc func onShowResultsButtonTapped() {
+        print("call onShowResultsButtonTapped")
+    }
 }
 
 extension ASCFiltersViewController: UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
```

Comment:
Need to rewrite the code below. It is necessary to control the appearance at the view level and not at the controller level. Change view states through properties. Check the appearance on the light and dark theme of the system.

Reply:
Need to fix a **bug** while scrolling